www/templates/html/Peoplefinder.tpl.php

```diff
@@ -22,7 +22,7 @@
 $page->addStyleDeclaration("#dcf-mobile-toggle-menu {display: none!important}");
 
 $page->doctitle = '<title>Directory | University of Nebraska–Lincoln</title>';
-$page->titlegraphic = 'Directory';
+$page->titlegraphic = '<a class="dcf-txt-h5" href="/">Directory</a>';
```

Comment:
You should be able to do UNL_Peoplefinder::getURL() or $controller->getURL() rather than "/"

Reply:
Updated...